src/core/tracker.cpp

```diff
@@ -306,6 +306,9 @@ void Tracker::processPositionInformation( const GnssPositionInformation &positio
   if ( !mIsActive && !mIsReplaying )
     return;
 
+  if ( mFilterAccuracy && positionInformation.accuracyQuality() == GnssPositionInformation::AccuracyBad )
+    return;
```

Comment:
Nice, much cleaner  -- you also need to add this logic in Tracker::replayPositionInformationList.

Also, I think we need to tolerate AccuracyUndetermined here. Because I could see a situation where filterAccuracy is set to true but the positioning itself isn't determining accuracy. In that case we'd want to accept them to avoid weird UX.